src/raftstore/store/worker/read.rs

```diff
@@ -1,8 +1,8 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
 
-use std::cell::RefCell;
+use std::cell::{Cell, RefCell};
```

Comment:
maybe we should move `read.rs` out of work dir. It is not a worker anymore.

Reply:
Let's do it later.